### DISCUSSION TITLE
Is there a way to bind to double tapping?

### DISCUSSION THREAD

**Discussion by TxHawks** [kasper/phoenix#334] [Q&A]
Is there a way to bind an action to when a key is double-tapped within a certain time?

I'd like to double tap the `alt` key to bring up my terminal app.

**Comment by kasper** [comment]
@TxHawks Hi! Keyboard shortcuts don’t allow binding to just a modifier key. There has been some asks of also supporting raw key events in Phoenix, this would likely be possible with that. However then you couldn’t exclusive react to the key press, but other apps would also react.

You can do something like this:

```js
Key.on(
  'e',
  ['control', 'shift'],
  _.after(2, () => {
    console.log('Function called after 2 invokes.');
  }),
);
```

**Reply by TxHawks** [reply]
Thanks for the reply @kasper 

What do you mean by "you couldn’t exclusive react to the key press, but other apps would also react"?

**Reply by kasper** [reply]
Keyboard shortcuts are registered to an app, first-come first-served basis. No other app can register the shortcut when it’s exclusive registered globally to an app.

Listening to keyboard events is possible, but then other apps will also react to the same key events. Example, let’s say you have an input selected and press the key <kbd>e</kbd> twice. We could listen to this in Phoenix, but also the input would also react to the same event. Currently there is no way of reacting to normal key events like this in Phoenix.

**Comment by TxHawks** [comment]
Thanks!
